Description: Remove Nikos Drakos' address from documentation
 In some early Debian revision already, Nikos Drakos' old web address (_not_
 the name) was removed from the documentation in favour of Ross Moore's one.
Author: Roland Stigge <stigge@antcom.de>

--- latex2html-2008-debian1.orig/latex2html.pin
+++ latex2html-2008-debian1/latex2html.pin
@@ -14071,7 +14071,7 @@ sub default_textohtmlinfopage {
 	, "<A HREF=\"$TEX2HTMLADDRESS\"><STRONG>LaTeX</STRONG>2<tt>HTML</tt></A>"
 	, " translator Version $TEX2HTMLVERSION\n"
 	, "<P>Copyright &#169; 1993, 1994, 1995, 1996,\n"
-	, "<A HREF=\"$AUTHORADDRESS\">Nikos Drakos</A>, \n"
+	, "Nikos Drakos, \n"
 	, "Computer Based Learning Unit, University of Leeds.\n"
 	, "<BR>Copyright &#169; 1997, 1998, 1999,\n"
 	, "<A HREF=\"$AUTHORADDRESS2\">Ross Moore</A>, \n"

